verify-version.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# To publish a build, use the Makefile
+# make publish
+# which will call out to this script
+
+# Use maven-help-plugin to get the current project.version
+NEW_VERSION=`mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v '\['`
+export NEW_VERSION
+
+# Use maven-help-plugin to update the project.version
+mvn versions:set -DnewVersion=$NEW_VERSION -DgenerateBackupPoms=false
+
+# Update the Version.java class
```

Comment:
You're still updating the version here?

Reply:
I think it's idempotent? In case someone didn't use `make inc-version`, and updated the mvn pom versions manually, this will make sure `Version.java` is updated